R/promptAndRun.R

```diff
@@ -15,27 +15,28 @@
 #' }
 #'
 promptAndRun <- function(mydir = ".", user = NULL, daysback = 3) {
```

Comment:
use colors everywhere, except when you run `-ab` or `-cb` as command line arguments, so the `b` serves as `black` option.